### PR TITLE
Add City switch button in Search #129

### PR DESCRIPTION
**Pull request by nicolasleger**
screen shot 2017-04-17 at 15 18 47

To test :

1. Go on http://localhost:3000/recherche?utf8=%E2%9C%93&search%5Bfrom_city%5D=Paris&search%5Bfrom_lon%5D=2.333333&search%5Bfrom_lat%5D=48.866667&search%5Bto_city%5D=Dijon&search%5Bto_lon%5D=5.016667&search%5Bto_lat%5D=47.316667&search%5Bdate%5D=&commit=Rechercher
2. Click on the switch button then search
3. You must see the parameters in URL and input values inverted

### CODE REVIEW

**Comment by nicolasleger** [comment]
@Poussinou @thb @webplate anyone to check and approve this PR please?

**Comment by webplate** [comment]
Hello @nicolasleger and @Poussinou !
Good news !
I've met @thb and he has given me the ability to merge on master and deploy for testing on http://covoli-dev.scalingo.io :)
I can help him now to review/merge PRs !

Many PRs were merged yesterday and I want to merge this one too as its great and important. It's important but it is also an interface change on the main page. More or less the first thing you see, so I think these layouts should be improved : 

Small screen

switch button could be smaller and on the side.
Wide screen

"Partez" button could be vertically aligned with "à" field.

What's your opinion @nicolasleger ?

**Comment by nicolasleger** [comment]
Good news !
Thanks, yes I approved your better design, I will try to do it.

**Comment by nicolasleger** [comment]
@webplate I updated the button style, you can see the result on https://covoiturage-libre.herokuapp.com/

File: app/assets/stylesheets/_home.scss [hunk 1]
```diff
@@ -137,5 +137,6 @@ $bgFeatureBlock: #fffbf2 !default;
 .btn-search-inverse-cities {
   line-height: 100%;
   margin: 0;
+  padding: 13px 14px;
   width: 100%;
 }
```
Comment (webplate) [inline app/assets/stylesheets/_home.scss]: Actually this makes the switch button too big. It's not the main attraction here.
oversize

I like the short : 
.btn-search-inverse-cities {
   line-height: 100%;
 }

Comment (nicolasleger) [inline app/assets/stylesheets/_home.scss]: @webplate feel free to edit my PR, I'm not a CSS expert.

**Comment by webplate** [comment]
OK @nicolasleger 
I have made some modifications in a personal branch:
[feat/129_switch_button](https://github.com/webplate/covoiturage-libre/tree/feat/129_switch_button)

It's not perfect but it's functional and the button is smaller on all screen sizes and pages.
Do I merge it directly with master and remove this PR or do you want to merge it first with your branch feature/129_add_city_switch_button_search ?

Or is there a way to edit your PR directly ?
I have never done that with Git yet...

Always nice to work with you :)

**Comment by Poussinou** [comment]
@webplate whaaaaaa!!! A lot of work around here :) I know you can push the changes on scalingo, can you ping me if you do it. I'll ask for testers on Facebook and slack to test it ;) can you also pit this changes on production or not?

**Comment by webplate** [comment]
Hello @Poussinou :) Yes I'm back for a little work ! The scalingo test site is down and I don't know why... @thb holds the answer I hope but is not answering right now. Can you join him ?
I cannot push to production, only to master and to scalingo but only if it's operational.
Thanks for the potential testing crowd ! That's what we'll need :)

**Comment by webplate** [comment]
@nicolasleger So I guess I can merge from my branch https://github.com/webplate/covoiturage-libre/tree/feat/129_switch_button directly and then close this PR.

**Comment by nicolasleger** [comment]
As you wish to proceed

**Pull request merged**